### PR TITLE
VIH-7315 Extract on-the-day components for translation

### PR DESCRIPTION
**Pull request by tremolo64**
### JIRA link (if applicable) ###
VIH-7315

### Change description ###
Extract on-the-day components for translation

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/judge-hearing-list/judge-hearing-table.component.html [hunk 1]
```diff
@@ -45,47 +45,47 @@
               <span *ngIf="hearing.panelMembers.length > 0">
                 <p class="govuk-!-margin-0 govuk-!-padding-0">
                   <span id="panel-member-count-{{ hearing.id }}">
-                    {{ hearing.panelMembers.length }} Panel Member{{ hearing.panelMembers.length > 1 ? 's' : '' }}
+                    {{ hearing.panelMembers.length }} {{ hearing.panelMembers.length > 1 ? 'Panel Members' : 'Panel Member' }}
```
Comment (levibutler) [inline VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/judge-hearing-list/judge-hearing-table.component.html:48]: missing translate?

File: VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/judge-hearing-list/judge-hearing-table.component.html [hunk 2]
```diff
@@ -45,47 +45,47 @@
...
                   </span>
                 </p>
               </span>
               <span *ngIf="hearing.wingers.length > 0">
                 <p class="govuk-!-margin-0 govuk-!-padding-0">
                   <span id="winger-count-{{ hearing.id }}">
-                    {{ hearing.wingers.length }} Winger{{ hearing.wingers.length > 1 ? 's' : '' }}
+                    {{ hearing.wingers.length }} {{ hearing.wingers.length > 1 ? 'Wingers' : 'Winger' }}
```
Comment (levibutler) [inline VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/judge-hearing-list/judge-hearing-table.component.html:55]: translate?

Comment (levibutler) [reply VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/judge-hearing-list/judge-hearing-table.component.html:55]: there are a couple like this in this file :)

File: VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/judge-hearing-list/judge-hearing-table.component.html (hunk 1 above)
Comment (tremolo64) [reply VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/judge-hearing-list/judge-hearing-table.component.html]: Fixed

File: VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/judge-hearing-list/judge-hearing-table.component.html (hunk 2 above)
Comment (tremolo64) [reply VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/judge-hearing-list/judge-hearing-table.component.html]: Fixed

File: VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/camera-check/camera-check.component.html [hunk 3]
```diff
@@ -5,19 +5,19 @@
...
-    <h1 class="govuk-heading-xl">Was your camera working?</h1>
+    <h1 class="govuk-heading-xl">{{ 'camera-check.was-your-camera-working' | translate }}</h1>
 
     <form [formGroup]="form" class="govuk-form-group" (ngSubmit)="onSubmit()">
       <div [ngClass]="showError() ? 'govuk-form-group--error' : 'govuk-form-group'">
         <div *ngIf="showError()" class="govuk-error-message">
-          <span class="govuk-visually-hidden">Error:</span>
-          Select if you could see yourself in the camera window
+          <span class="govuk-visually-hidden">{{ 'camera-check.error' | translate }}</span>
+          {{ 'camera-check.select-if-you-could-see-yourself-in-the-camera-window' | translate }}
```
Comment (levibutler) [inline VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/camera-check/camera-check.component.html:14]: could we summarise? maybe
`camera-check.could-see-yourself`

File: VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/declaration/declaration.component.html [hunk 4]
```diff
@@ -5,20 +5,19 @@
...
   <div class="govuk-grid-column-two-thirds">
-    <h1 class="govuk-heading-xl">Declaration</h1>
+    <h1 class="govuk-heading-xl">{{ 'declaration.declaration' | translate }}</h1>
   </div>
 </div>
 <form [formGroup]="declarationForm" class="govuk-form-group" (ngSubmit)="onSubmit()">
   <div [ngClass]="declarationForm.invalid && submitted ? 'govuk-form-group--error' : 'govuk-form-group'">
     <div *ngIf="declarationForm.invalid && submitted" class="govuk-error-message">
-      <span class="govuk-visually-hidden">Error:</span> Please tick the box to confirm that you will not record the hearing in any way
+      <span class="govuk-visually-hidden">{{ 'declaration.error' | translate }}</span> {{ 'declaration.please-tick-the-box-to-confirm-that-you-will-not-record-the-hearing-in-any-way' | translate }}
```
Comment (levibutler) [inline VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/declaration/declaration.component.html:14]: could be shortened to 
declaration.confirm-not-recording

File: VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/declaration/declaration.component.html [hunk 5]
```diff
@@ -34,7 +33,7 @@ <h1 class="govuk-heading-xl">Declaration</h1>
             required
           />
           <label class="govuk-label govuk-checkboxes__label" for="declare">
-            I confirm that I will not record or take any images of this hearing
+            {{ 'declaration.i-confirm-that-i-will-not-record-or-take-any-images-of-this-hearing' | translate }}
```
Comment (levibutler) [inline VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/declaration/declaration.component.html:36]: shorten maybe

File: VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/hearing-rules/hearing-rules.component.html [hunk 6]
```diff
@@ -5,25 +5,25 @@
...
-        <li>can only drink water</li>
-        <li>must not eat</li>
-        <li>must only use your phone for contact with the video hearings team</li>
-        <li>must follow the judge’s instructions</li>
-        <li>are not allowed to smoke (including e-cigarettes)</li>
-        <li>must be alone unless you have the judge’s permission to have someone with you</li>
-        <li>must ask the judge's permission if you need to move away from your screen</li>
+        <li>{{ 'hearing-rules.can-only-drink-water' | translate }}</li>
+        <li>{{ 'hearing-rules.must-not-eat' | translate }}</li>
+        <li>{{ 'hearing-rules.must-only-use-your-phone-for-contact-with-the-video-hearings-team' | translate }}</li>
```
Comment (levibutler) [inline VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/hearing-rules/hearing-rules.component.html:22]: can we shorten?

File: VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/switch-on-camera-microphone/switch-on-camera-microphone.component.html [hunk 7]
```diff
@@ -6,39 +6,39 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-full">
     <div *ngIf="!userPrompted">
-      <h1 class="govuk-heading-xl">Use your camera and microphone</h1>
+      <h1 class="govuk-heading-xl">{{ 'switch-on-camera-microphone.use-your-camera-and-microphone' | translate }}</h1>
       <p class="govuk-body govuk-!-margin-top-9 govuk-!-margin-bottom-9">
-        Switch on your camera and microphone to watch the video.
+        {{ 'switch-on-camera-microphone.switch-on-your-camera-and-microphone-to-watch-the-video' | translate }}
```
Comment (levibutler) [inline VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/switch-on-camera-microphone/switch-on-camera-microphone.component.html:11]: can any of these be shortened?

File: VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/camera-check/camera-check.component.html (hunk 3 above)
Comment (tremolo64) [reply VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/camera-check/camera-check.component.html]: done

File: VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/declaration/declaration.component.html (hunk 4 above)
Comment (tremolo64) [reply VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/declaration/declaration.component.html]: Shortened

**Review by valentinabennett — DISMISSED** [review]

**Review by levibutler — APPROVED** [review]

**Pull request merged**